samples/scenarios/db_impersonation/connectionResolver.tdr

```diff
@@ -6,6 +6,7 @@
     </connection-builder>
     <connection-normalizer>
       <required-attributes>
+        <setImpersonateAttributes/>
```

Comment:
I believe you first need to update dev-2020.4 to include the changes like [this](https://github.com/tableau/connector-plugin-sdk/pull/633) made directly against master, before merging dev-2020.4 into master.

Reply:
Removed this line. As a part of https://github.com/tableau/connector-plugin-sdk/pull/659/commits/5391e46e10a64ac83dce211cca89b0a3254c9826 , have included the changes but had mistakenly manually kept this line